### DISCUSSION TITLE
Add seed into the save filename

### DISCUSSION THREAD

**Discussion by wutipong** [WASasquatch/was-node-suite-comfyui#163] [Q&A] [answered]
Is it possible to add seed value to the filename/output path? I was trying to convert the seed value to text and add it as a token but the value seems to be incompatible. 

Any idea to archive this?

**Comment by WASasquatch** [comment]
Convert int to number and number to text maybe?

**Comment by wutipong** [comment, marked as answer]
I can't find the int to number node. May be I overlook something.

Anyway I've got it working now using the random number node and use Number to Seed to seed the sampler. 

Thanks for the help!

**Reply by WASasquatch** [reply]
Probably simpler work around there.
